tests/test_databases_pymongo.py

```diff
@@ -0,0 +1,71 @@
+import os
+import tempfile
+
+import pytest
+
+from rxn.utilities.databases.pymongo import PyMongoSettings
+
+
+def _instantiate_client_and_check_server_info():
+    pymongo_settings = PyMongoSettings()
+    client = pymongo_settings.get_client()
+    server_info = client.server_info()
+    assert isinstance(server_info, dict)
+    assert "version" in server_info
+
+
+@pytest.fixture
+def mock_no_mongo_uri_env(monkeypatch):
+    """Delete environment variable RXN_MONGO_URI.
+
+    Needed to simulate environments with no MongoDB settings.
+    """
+    monkeypatch.delenv("RXN_MONGO_URI", raising=False)
+
+
+@pytest.fixture
+def mock_mongo_certificate_env(monkeypatch):
+    """If a certificate is provided as an environment variable store it in a file and configure the environment.
+
+    Needed to simulate environments with MongoDB TLS connections.
+    """
+    if "RXN_TLS_CA_CERTIFICATE" in os.environ:
+        directory = tempfile.mkdtemp(prefix="rxn-utilities-pymongo")
+        tls_ca_cert_filepath = os.path.join(directory, "ca.cert")
+        with open(tls_ca_cert_filepath, "w") as fp:
+            fp.write(os.environ["RXN_TLS_CA_CERTIFICATE"])
+        monkeypatch.setenv("RXN_TLS_CA_CERTIFICATE_PATH", tls_ca_cert_filepath)
+
+
+def test_get_pymongo_settings(mock_mongo_certificate_env):
+    pymongo_settings = PyMongoSettings()
+    assert isinstance(pymongo_settings, PyMongoSettings)
+    assert pymongo_settings.tls_ca_certificate_path == os.environ.get(
+        "RXN_TLS_CA_CERTIFICATE_PATH", None
+    )
+
+
+@pytest.mark.skipif(
+    "RXN_MONGO_URI" not in os.environ, reason="environment_not_configured_for_test"
+)
+def test_get_client():
+    _instantiate_client_and_check_server_info()
+
+
+@pytest.mark.skipif(
+    "RXN_MONGO_URI" not in os.environ, reason="environment_not_configured_for_test"
+)
+def test_get_client_with_tls_ca_cert(mock_mongo_certificate_env):
+    _instantiate_client_and_check_server_info()
+
+
+def test_get_pymongo_settings_with_no_mongo_uri(mock_no_mongo_uri_env):
+    pymongo_settings = PyMongoSettings()
+    assert isinstance(pymongo_settings, PyMongoSettings)
+    assert pymongo_settings.mongo_uri == os.environ.get("RXN_MONGO_URI", None)
```

Comment:
Could this be directly
```suggestion
    assert pymongo_settings.mongo_uri is None
```
?

Reply:
sure will update, it was a leftover :)

Reply:
regarding the comment below, correct that is the main difference.